static/js/components/ProfileImageUploader.js

```diff
@@ -32,6 +33,31 @@ const uploaderBodyHeight = (): number => (
 
 const imageError = err => <div className="img-error">{err}</div>;
 
+const dialogContents = (
+  updatePhotoEdit,
+  photo,
+  startPhotoEdit,
+  setPhotoError,
+  inFlight
+) => {
+  if ( inFlight ) {
+    return <div
+      className="photo-active-item dashed-border"
+      style={{height: uploaderBodyHeight()}}
+    >
+      <Spinner singleColor />
```

Comment:
I think this spinner is supposed to be the MIT red, not blue, according the mocks in #2019.